iped-app/resources/config/conf/ParsingTaskConfig.txt

```diff
@@ -19,7 +19,7 @@ parseCorruptedFiles = true
 
 # Minimum timeout (seconds) while parsing files. To this baseline will be added the result of timeOutPerMB * file size in MB (see next parameter).
 # After the timeout, the file's raw strings will be indexed.
-timeOut = 180
+timeOut = 18000
```

Comment:
This can have undesired effects other unrelated modules, this is really needed? If timeouts are being caused by an internal IPED parser (not a Tika one), we can reset the timeout counter from inside the parser from time to time just writing an empty string in the content handler.

Reply:
This was pushed by mistake. It was for debugging purpose.